sdk/core/core-amqp/.eslintrc.json

```diff
@@ -1,7 +1,9 @@
 {
   "plugins": ["@azure/azure-sdk"],
-  "extends": ["../../.eslintrc.old.json", "plugin:@azure/azure-sdk/recommended"],
-  "parserOptions": {
-    "createDefaultProgram": true
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+    "@typescript-eslint/no-unused-vars": "off",
+    "@azure/azure-sdk/ts-no-namespaces": "off",
+    "no-inner-declarations": "off"
```

Comment:
Curious where we were hitting this and whether or not we should just disable this rule globally. We are targeting ES6, so nested function declarations should be allowed. Alternatively, we could convert them to arrow functions.

Reply:
I think we need to get at least @xirzec involved before we take a direction to either.

Reply:
IIRC this rule only applies to function declarations that leak out of block scopes.

This rule can be left on and anonymous functions that are scoped with let/const can be used instead without ambiguity.

Reply:
Mr Jeff, I'll get in touch with you today so that you can help me understand the direction you're suggesting.

Reply:
Awesome! So I had this meeting with @xirzec and we agreed on changing the namespaces to object const! I'll push that today.

Reply:
Pushed!